dist/api_data.py

```diff
@@ -364,11 +364,6 @@ def __ge__(self, other):
         allowed value are both one-tenth the size of a newly split internal
         page''',
         min='0'),
-    Config('key_gap', '10', r'''
-        the maximum gap between instantiated keys in a Btree leaf page,
-        constraining the number of keys processed to instantiate a
-        random Btree leaf page key''',
-        min='0', undoc=True),
```

Comment:
The `key_gap` configuration string is WiredTiger only, it's never been used by MongoDB (afaik).

@agorrod, flagging you for a ruling, is it OK to remove the `key_gap` configuration string? It's been undocumented for at least 5 years and is no longer used.

Reply:
I'm not saying MongoDB actively uses `key_gap` but the item does appear in the current metadata string with its default value and the code in `btree_conf` does an unconditional (for row store) `wt_config_gets` on that configuration component. Will a "new" metadata file without that value in row-store tables that ran in an upgraded database work/not error with an older executable?

Reply:
@sueloverso, that sounds possible to me, nice catch!

@agorrod, is @sueloverso correct, should I put the `key_gap` field back into the metadata?

Reply:
@sueloverso, I'm pretty confident you're correct in your analysis, so I've just put the `key_gap` field back into the metadata.

Would you please do the final review, and I'll go ahead and merge. Thanks!

Reply:
My only (very) minor comment now is to make this and `huffman_key` consistent in their "no longer supported" verbiage. I don't care which wording is used but it would be nice if they were the same or very similar.